docs/linking_events.md

```diff
@@ -0,0 +1,97 @@
+# Linking Events
+
+## Linking single event
+
+To create a link on an existing event, you need to have a stream where you want to link that event to and you need to have an event, fetched from the database:
```

Comment:
`you need an event and a destination stream where you want to link the event` or maybe Nico will propose sth better

Reply:
k, will improve